plasmapy/simulation/particle_tracker.py

```diff
@@ -285,9 +285,31 @@ def save_now(self):
 
 
 class ParticleTracker:
-    """
-    General particle tracer.
+    r"""A many body particle tracker for electric and magnetic fields with no particle-particle interactions.
+
+    Particles are created and pushed through calculated E and B fields. These fields are specified as part of a grid
```

Comment:
created -> instantiated 

Calculated -> "a grid of provided E&B fields" We want to differentiate from trackers that calculate the fields from specified charge locations, or from the particles themselves.